examples/flutter_gallery/lib/gallery/app.dart

```diff
@@ -70,6 +70,12 @@ class _GalleryAppState extends State<GalleryApp> {
     model = AppStateModel()..loadProducts();
   }
 
+  @override
+  void reassemble() {
+    _options = _options.copyWith(platform: defaultTargetPlatform);
```

Comment:
Without this, the options does not get updated for new platforms on a reassemble